Amazon_Web_Services/RDS/Log-Collection/rds.py

```diff
@@ -48,13 +48,13 @@
         response=client.download_db_log_file_portion(
             DBInstanceIdentifier=dBInstanceIdentifier,
             LogFileName=logFile['LogFileName'],
-            Marker=readMarker,
+            Marker=readMarker
         )
         if len(response['LogFileData']) > 0:
-            logLines=response['LogFileData'].strip().splitlines() # LogFileData sends all entries as a single string. Split it up into a list to be able to append text to the start
-            for entry in logLines:
-                print("["+ dBInstanceIdentifier + " " + logFile['LogFileName'] + "] " + entry)
-        readState[logFile['LogFileName']]=response['Marker']
+            r = response[ 'LogFileData' ].strip() 
```

Comment:
splitlines was causing my instance problems identifying the boundary between log entries.

Reply:
@danreichert @stevenchamales Whats the word on this change? (as soon as we hash it out, I can merge ;)